boards/xtensa/up_squared_adsp/bootloader/boot_ldr.x

```diff
@@ -0,0 +1,208 @@
+OUTPUT_ARCH(xtensa)
+MEMORY
```

Comment:
I guess this does not need to be pre-processed for Zephyr ?

Reply:
Probably not - I think it was taken from SOF largely as-is. @finikorg any comment on this?

Reply:
We have "bootloader" code, similar to SOF's one. That script places objects to the right places.

Reply:
ok, FWIW SOF pre-processes linker scripts with some headers to align the sections etc. I'm good if you guys are happy.